File: app/src/main/java/de/xikolo/utils/FileProviderUtil.java
package de.xikolo.utils;

import android.net.Uri;

import java.io.File;

import androidx.core.content.FileProvider;
import de.xikolo.App;
import de.xikolo.BuildConfig;

public class FileProviderUtil extends FileProvider {

    public static Uri getUriForFile(File file) {
        return getUriForFile(App.getInstance(), BuildConfig.APPLICATION_ID + ".fileprovider", file);
    }

}

File: app/src/main/java/de/xikolo/utils/FileProviderUtil.kt
package de.xikolo.utils

import android.net.Uri
import androidx.core.content.FileProvider
import de.xikolo.App
import de.xikolo.BuildConfig
import java.io.File

class FileProviderUtil : FileProvider() {

    companion object {
        fun getUriForFile(file: File): Uri {
            return getUriForFile(App.instance, BuildConfig.APPLICATION_ID + ".fileprovider", file)
        }
    }

}
